integration-commons/src/main/java/org/symphonyoss/integration/BaseIntegration.java

```diff
@@ -125,16 +126,20 @@ public void registerUser(String integrationUser) {
 
     Application application = properties.getApplication(integrationUser);
 
+    if (application == null) {
+      throw new CertificateNotFoundException("Certificate file unknown");
```

Comment:
Why is this a certificate exception?  Also, this error message doesn't give enough info to solve the problem.  For example, doesn't tell which application is having a problem.